### ISSUE TITLE
Redmi 4X (santoni) kernel source - please

### ISSUE THREAD

**Issue by Akkowicz** [MiCode/Xiaomi_Kernel_OpenSource#683]
This model is pretty popular, a lot of people love your devices, but just can't stand MIUI.

**Comment by saikatm** [comment]
@Akkowicz Yes, agree people and lot of devs are waiting for the Kernel sources to be released for **Santoni**, There are no decent amount of custom ROMs because of this.

**Comment by thejayyadav** [comment]
Option for using other ROMs make it more popular
